server/compute/diffexp_cxg.py

```diff
@@ -27,18 +27,33 @@ def diffexp_ttest(adaptor, maskA, maskB, top_n=8, diffexp_lfc_cutoff=0.01, arr="
       https://en.wikipedia.org/wiki/Bonferroni_correction
 
     :param adaptor: DataAdaptor instance
-    :param maskA: observation selection mask for set 1
-    :param maskB: observation selection mask for set 2
+    :param setA: observation selector (mask or list) for set 1
+    :param setB: observation selector (mask or list) for set 2
     :param top_n: number of variables to return stats for
     :param diffexp_lfc_cutoff: minimum
     :param arr: the tdb array to read
+    :param selector_lists: if True, the selectors are presumed to be masks of length n_obs; else, lists of obs indices
     absolute value returning [ varindex, logfoldchange, pval, pval_adj ] for top N genes
     :return:  for top N genes, {"positive": for top N genes, [ varindex, foldchange, pval, pval_adj ],
               "negative": for top N genes, [ varindex, foldchange, pval, pval_adj ]}
```

Comment:
Not changed on this PR, but I can't quite parse the above; should it be as follows?

```suggestion
    :return:  for top N genes, {"positive": [ varindex, foldchange, pval, pval_adj ], "negative": [ varindex, foldchange, pval, pval_adj ]}
```